README.md

```diff
@@ -0,0 +1,245 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[hello\_microverse\_code\_reviewer\] Hello Microverse Code Reviewer](#-hello_microverse_code_reviewer-hello-microverse-code-reviewer)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello_microverse_code_reviewer] <a name="about-project">Hello Microverse Code Reviewer</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[hello_microverse_code_reviewer]** is a basic project to learn how the code review process will go.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```

Comment:
Also omit this part just a i suggested above. lets keep the README clean, which you'll agree with me is the best 😃